replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+  distinct(prop.tradeoff, prop.location, CI.tradeoff, CI.location)
+#computes new variables for proportion of answers, standard errors, and 95% confidence intervals for higher income group
+
+title = factor(c("trade-offs", "location"), levels = c("trade-offs", "location"))
+values = c(figure1alow$prop.tradeoff, figure1alow$prop.location, figure1ahigh$prop.tradeoff, figure1ahigh$prop.location)
+income = c("lower income", "lower income", "higher income", "higher income")
+CIs = c(figure1alow$CI.tradeoff, figure1alow$CI.location, figure1ahigh$CI.tradeoff, figure1ahigh$CI.location)
+fig1a.table <- data.frame(title, values, income, CIs)
+income.factor <- factor(fig1a.table$income, levels = c("lower income", "higher income"))
+#figure 1a data frame (+ income.factor reverses the default position dodge of the bars)
+
+figure1a <- fig1a.table %>%
+  ggplot(aes(x = title, y = values, fill = income.factor)) + geom_bar(stat = "identity", position = position_dodge(), colour = "black") + ylab("Proportion Endorsing Reason") + xlab(NULL) + geom_errorbar(aes(ymin = values - CIs, ymax = values + CIs), width = 0.2, position = position_dodge(0.9)) + theme_bw() + scale_fill_grey(start = .3, end = .85) + ggtitle("Figure 1a - the proportion of lower- and higher-income participants\nciting trade-off and location considerations in Study 1a\n(error bars represent 95% confidence interval)") + theme(panel.border = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))
+#produces figure 1a
+
+print(figure1a)
+#displays figure 1a
+```
+
+### Study 1b
+
+> In Study 1b, there were 151 participants (mean age = 32.9 years; 76 females, 75 males; median household size = 3 people, median household income = $45000).  In Study 1c, there were 604 participants (mean age = 29.9 years; 246 females, 358 males; median household size = 2 people; median household income = $45,000).  Study 1d was conducted with a large, representative sample drawn from Survey Sampling International's panel of U.S respondents.  There were 2,015 participants (mean age = 39.1 years; 1,022 females, 993 males; median household size = 3 people; median household income = $55,000).  In Study 1b, participants read the beer-on-the-beach scenario and indicated their WTP for the beer.  Studies 1c and 1d directly replicated Study 1b.  Because WTP was elicited with an open-ended scale, some participants named unreasonably high prices.  We describe our exclusion rules for these responses in the next section.  These rules were determined before we conducted the significance tests.
+
+```{r echo = FALSE}
+
+#read in the data for 1b using the spss file and csv file - the warning that comes up is okay - the data imports cleanly.
+suppressMessages(suppressWarnings(raw_1b_spss <- read.spss("framing/study_1b.sav", to.data.frame = TRUE)))
```

Comment:
Nice note about warning message! Maybe reference some wording in the message though (just in case future versions of R produce a different difficulty that shouldn't be ignored).